status/status_test.go

```diff
@@ -295,10 +293,10 @@ func TestStatus_ErrorDetails_Fail(t *testing.T) {
 			FromProto(&spb.Status{
 				Code: int32(cpb.Code_CANCELLED),
 				Details: []*apb.Any{
-					{
-						TypeUrl: "",
-						Value:   []byte{},
-					},
+					// {
```

Comment:
commented out?

Reply:
Let me remove that. This check will not work with the new proto package, but I think that is actually ok